core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -232,6 +234,36 @@ protected function doReplace($strBuffer, $blnCache)
 						break;
 					}
 
+					if ($keys[0] == 'LNG' && \count($keys) == 2)
+					{
+						trigger_deprecation('contao/core-bundle', '4.12', 'Using the label::LNG insert tag has been deprecated and will no longer work in Contao 5.0.');
```

Comment:
The alternative is missing :)

Reply:
Removed deprecation in 0961714d9b9ef9324653581597d7d62782d4aa1a